lib/src/configuration.dart

```diff
@@ -51,11 +51,11 @@ class Configuration {
   /// The Apps release name, defaults to 'name@version+buildNumber' from pubspec or set via env. var. SENTRY_RELEASE
   /// Example, name: 'my_app', version: 2.0.0+1, in this case the release is my_app@2.0.0+1
   /// This field has precedence over the [name] from pubspec
-  /// If this field has a build number, it has precedence over the [version]'s buid number from pubspec
+  /// If this field has a build number, it has precedence over the [version]'s build number from pubspec
   late String? release;
 
-  /// The Apps dist/build number, defaults to the build number after the '+' char from [version]'s pubspec or ser via env. var. SENTRY_DIST
-  /// Example, version: 2.0.0+1, in this case the build number is 1
+  /// The Apps dist/build number, taken from pubspec dist or SENTRY_DIST env. variable
```

Comment:
I assume sentry-defines should also be included here as soon as it's merged right?

Reply:
Yes, probably best to do this from the sentry-defines PR